tests/plugins/uploadwidget/uploadwidget.js

```diff
@@ -156,6 +156,49 @@
 			} );
 		},
 
+		// (#1454)
+		'test onAbort': function() {
```

Comment:
This test passes also without the fix. The source of the issue is simple: `onAbort` does not fire only when widget is removed from the editor (due to https://github.com/ckeditor/ckeditor-dev/blob/6b32b1b109d85ff998df79861f59e78c0add17aa/plugins/uploadwidget/plugin.js#L256 ). This test does not remove widget from the editor, therefore it doesn't test anything. The same is true for the other test as well.

This discovery also made me wondering, if the fix could be even more simplified, e.g. moving handling of abort to the beginning of the `update` listener? Not sure about it however.

Reply:
You are right. When refactoring test I inadvertently removed code responsible to test the issue.